lib/handlers/networkHandler.js

```diff
@@ -78,7 +78,28 @@ const setNetworkEmulation = async (networkType) => {
   });
 };
 
+const setUserAgent = async (deviceEmulate) => {
+  await network.setUserAgentOverride({
+    userAgent: deviceEmulate.userAgent,
+  });
+};
+
+const setCookie = async (options) => await network.setCookie(options);
+
+const getCookies = async (options) => {
+  return await network.getCookies(options);
+};
+
+const clearBrowserCookies = async () => await network.clearBrowserCookies();
+
+const deleteCookies = async (options) => await network.deleteCookies(options);
+
```

Comment:
does this belong to network handler or browser handler?

Reply:
they belong to network handler as they are methods of Network domain in CDP ... https://chromedevtools.github.io/devtools-protocol/tot/Network/